advanced/Scripts/update.sh

```diff
@@ -26,8 +26,24 @@ source "/opt/pihole/COL_TABLE"
 
 # is_repo() sourced from basic-install.sh
 # make_repo() sourced from basic-install.sh
-# update_repo() source from basic-install.sh
+# update_repo() sourced from basic-install.sh
 # getGitFiles() sourced from basic-install.sh
+# PIHOLE_DNS_1 sourced from basic-install.sh
+
+
+CheckInternetConnection() {
+  # Checks internet connection
+  # Pings DNS choosed by user during installation
+  
+  echo -e "  ${INFO} Checking internet connection..."
+
+  if ping -q -W 1 -c 2 "${PIHOLE_DNS_1}" > /dev/null 2>&1; then
+    echo -e "  ${TICK} Internet connection available"
+  else
+    echo -e "  ${COL_LIGHT_RED}Internet connection not available. Connect your device to internet${COL_NC}" 
```

Comment:
This could just be a `${CROSS}` instead, since it exits out of the script the user doesn't need the entire line to be in red.